### ISSUE TITLE
How can a data property be restricted to apply only to a specific type of statement?

### ISSUE THREAD

**Issue by strawberrylunar** [w3c/rdf-star#284]
For example, how would I assign a 'startedTime' property exclusively to all triples where the predicate is 'married'?" How to define "startedTime" in this case?

**Comment by TallTed** [comment]
`ex:Person ex:age ex:Literal` is a poor choice, as it will be accurate (or "certain") for *at most* one year.

**Comment by strawberrylunar** [comment]
> `ex:Person ex:age ex:Literal`_是一个糟糕的选择，因为它最多_在一年内是准确的（或“确定的”） 。

Yes, I admit it. I'm looking for a right answer.

**Comment by strawberrylunar** [comment]
> `ex:Person ex:age ex:Literal` is a poor choice, as it will be accurate (or "certain") for _at most_ one year.

Yes, I admit it. I'm looking for a right answer.

**Comment by strawberrylunar** [comment]
> `ex:Person ex:age ex:Literal` is a poor choice, as it will be accurate (or "certain") for _at most_ one year.

Yes, I admit it, that's why I'm looking for a right answer. I would appreciate it if you can help me !
